pom.xml

```diff
@@ -65,7 +65,7 @@
              make sure you have a look at the AmazonServices member section below
              and also at src/main/resources/xslt/amazon-services/test-pom.xsl
              as they might require adjustments. -->
-        <quarkus-amazon-services.version>2.16.2</quarkus-amazon-services.version>
+        <quarkus-amazon-services.version>2.18.0</quarkus-amazon-services.version>
```

Comment:
Should't this really be 2.18.1 as stated in the PR title? 2.18.0 is lacking the test jars.

Reply:
Yes it should, did I miss to force push ?